lib/src/folding/foldable_block.dart

```diff
@@ -5,31 +5,33 @@ import 'package:equatable/equatable.dart';
 import 'foldable_block_type.dart';
 
 class FoldableBlock with EquatableMixin {
-  final int startLine;
-  final int endLine;
+  final int firstLine;
+  final int lastLine;
```

Comment:
'end' traditionally refers to the first index after the range while 'first' and 'last' refer to inclusive ranges. This range is inclusive on both ends.